SwiftGit2Tests/Fixtures/Fixtures.swift

```diff
@@ -47,12 +53,20 @@ final class Fixtures {
 	}
 	
 	func unzipFileAtURL(fileURL: NSURL, intoDirectoryAtURL directoryURL: NSURL) {
+#if os(OSX)
+
 		let task = NSTask()
 		task.launchPath = "/usr/bin/unzip"
 		task.arguments = [ "-qq", "-d", directoryURL.path!, fileURL.path! ]
-		
+            
 		task.launch()
 		task.waitUntilExit()
+
+#else
+
+		assertionFailure("Tests not supported on iOS yet")
```

Comment:
I ran into this issue with Gift as well: https://github.com/modocache/Gift/commit/f1c73dfdc3c042d2e6027bb0be321787db792511

Since `system()` and `NSTask` aren't available, I used `zlib`--or, more specifically, I used @soffes's [SSZipArchive], which wraps `zlib` with a friendlier API.